### PR TITLE
Ports "Ports "Revives "Adds sound effects to surgeries"" and others" and "Adds medical sounds"

### PR DESCRIPTION
**Pull request by nmajask**
## About The Pull Request

Ports two (2) PRs from yogs and bay (technically they are both ports from yogs but shhhh), those being ["Ports "Revives "Adds sound effects to surgeries"" and others" #14390](https://github.com/yogstation13/Yogstation/pull/14390) by [some idiot](https://github.com/nmajask) and ["Adds medical sounds" #21268](https://github.com/Baystation12/Baystation12/pull/21268) by [comma](https://github.com/comma). In short, adds surgery sounds to most surgery steps and when applying bruise packs/ointment. Also adds sounds for starting surgeries for some nice feedback for when a surgery is/isn't started. May be a bit loud, so I can make it quieter if needed. Bone sounds are from [freesound.org](https://freesound.org/people/carlito62/sounds/176745/).

## Why It's Good For The Game

More sounds good, and makes surgery a little more involving than just clicking on the horizontal person.

## Changelog
:cl:  maxspells, TheBoondock, MrMelbert, comma, nmajask
soundadd: added sounds to surgery steps
soundadd: added sound effects to a bunch of medical items
spellcheck: fixed revive surgery saying "shock body" instead of "shock brain"
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -18,6 +18,8 @@
 	var/other_delay = 0
 	var/repeating = FALSE
 	var/experience_given = 1
+	/// Shiptest Edit - Sound/Sounds to play when this is applied
```
Comment (Fikou) [inline code/game/objects/items/stacks/medical.dm]: We don't do this anymore.

**Review by Fikou — DISMISSED** [review]
remove all shiptest edits and move the files out of the whitesands folder

File: code/game/objects/items/stacks/medical.dm (hunk 1 above)
Comment (nmajask) [reply code/game/objects/items/stacks/medical.dm]: Are you talking about the Shiptest Edit or autodocing?

Comment (Fikou) [reply code/game/objects/items/stacks/medical.dm]: Yes

File: code/modules/surgery/organ_manipulation.dm [hunk 2]
```diff
@@ -80,6 +82,8 @@
 /datum/surgery_step/manipulate_organs/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	manipulated_organ = null
 	if(istype(tool, /obj/item/organ_storage))
+		preop_sound = initial(preop_sound)
+		success_sound = initial(success_sound)
 		if(!tool.contents.len)
 			to_chat(user, "<span class='warning'>There is nothing inside [tool]!</span>")
 			return -1
```
Comment (ZephyrTFA) [inline code/modules/surgery/organ_manipulation.dm:89]: I know this isn't your code, but what does -1 even do here. This should probably be a define

**Review by MarkSuckerberg — APPROVED** [review]
Sorry we've been dragging our feet on this... I'll get it testmerged.

**Comment by nmajask** [comment]
> Sorry we've been dragging our feet on this... I'll get it testmerged.

all good, as long as its merged eventually

**Pull request merged**